src/v/config/configuration.cc

```diff
@@ -102,6 +102,12 @@ configuration::configuration()
       "Milliseconds for raft leader heartbeats",
       required::no,
       std::chrono::milliseconds(150))
+  , raft_heartbeat_timeout_ms(
```

Comment:
i don't get this. 

i thought we simply drop the RPC if there is an ongoing `append_entries` or an ongoing `heartbeat` RPC

what is this timeout for then?

Reply:
We can not just drop it. It was already sent. We have to somehow make this RPC to terminate. Dropping at receiver is not enough. When packets would be dropped f.e. by firewall it wouldn't terminate.

Reply:
after live review, i get this. 

this is basically needs to be bigger than the heartbeat **send***-rpc timeout so we can prevent backlog build up on the RPC **received**-side and not flood the wire.